<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainPage</class>
 <widget class="QWidget" name="MainPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
border-image: url(':/ui/themes/default/main_page.png');
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget_main">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>260</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="pageMain">
    <widget class="QStackedWidget" name="stackedWidget_list">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>261</width>
       <height>431</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>261</width>
       <height>421</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextBrowser{
	background-color: transparent;
}</string>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="page_messageList">
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <widget class="QScrollArea" name="scrollArea_messages">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>261</width>
         <height>411</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="Widget_messageList">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>261</width>
          <height>411</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>411</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayoutMsg">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTextBrowser" name="textBrowser_messages">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>261</width>
             <height>411</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>楷体</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QTextBrowser {
background-color: transparent;
color:rgb(255, 255, 255)
}
QScrollArea{
border:0px solid;
border-radius: 5px;
}

QScrollBar:vertical{
width: 10px;
border:0px solid;
border-radius: 5px;
margin: 0px,0px,0px,0px;
background: rgb(103, 103, 153);
}

QScrollBar:vertical:hover{
width: 10px;
border:0px solid;
margin: 0px,0px,0px,0px;
}
QScrollBar::handle:vertical{
width:10px;
background:#c0ddee;
border-radius: 5px;
height: 50px;
}

QScrollBar::handle:vertical:hover{
background:#c0ddee;
border-radius: 5px;
}

QScrollBar::add-line:vertical{
height:20px;
transparent;
subcontrol-position:bottom;
border:0px solid;
border-radius: 5px;
}
QScrollBar::sub-line:vertical{
height:20px;
transparent;subcontrol-position:top;
border:0px solid;
border-radius: 5px;
}
QScrollBar::add-page:vertical{
 #e1f0fa;
border:0px solid;border-radius: 5px;
}

QScrollBar::sub-page:vertical{
border:0px solid
;border-radius: 5px;
}
QScrollBar::up-arrow:vertical{
border:0px solid;
border-radius: 3px;
}

QScrollBar::down-arrow:vertical {
border:0px solid;
border-radius: 3px;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="page_friendList">
      <property name="styleSheet">
       <string notr="true">QScrollArea{
border:0px solid;
border-radius: 5px;
}

QScrollBar:vertical{
width: 10px;
border:0px solid;
border-radius: 5px;
margin: 0px,0px,0px,0px;
background: rgb(103, 103, 153);
}

QScrollBar:vertical:hover{
width: 10px;
border:0px solid;
margin: 0px,0px,0px,0px;
}
QScrollBar::handle:vertical{
width:10px;
background:#c0ddee;
border-radius: 5px;
height: 50px;
}

QScrollBar::handle:vertical:hover{
background:#c0ddee;
border-radius: 5px;
}

QScrollBar::add-line:vertical{
height:20px;
transparent;
subcontrol-position:bottom;
border:0px solid;
border-radius: 5px;
}
QScrollBar::sub-line:vertical{
height:20px;
transparent;subcontrol-position:top;
border:0px solid;
border-radius: 5px;
}
QScrollBar::add-page:vertical{
 #e1f0fa;
border:0px solid;border-radius: 5px;
}

QScrollBar::sub-page:vertical{
border:0px solid
;border-radius: 5px;
}
QScrollBar::up-arrow:vertical{
border:0px solid;
border-radius: 3px;
}

QScrollBar::down-arrow:vertical {
border:0px solid;
border-radius: 3px;
}</string>
      </property>
      <widget class="QScrollArea" name="scrollArea_friends">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>261</width>
         <height>421</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="Widget_friendList">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>261</width>
          <height>421</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>411</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayoutFrindList">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="page_groupList">
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <widget class="QScrollArea" name="scrollArea_Group">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>261</width>
         <height>421</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QScrollArea{
border:0px solid;
border-radius: 5px;
}

QScrollBar:vertical{
width: 10px;
border:0px solid;
border-radius: 5px;
margin: 0px,0px,0px,0px;
background: rgb(103, 103, 153);
}

QScrollBar:vertical:hover{
width: 10px;
border:0px solid;
margin: 0px,0px,0px,0px;
}
QScrollBar::handle:vertical{
width:10px;
background:#c0ddee;
border-radius: 5px;
height: 50px;
}

QScrollBar::handle:vertical:hover{
background:#c0ddee;
border-radius: 5px;
}

QScrollBar::add-line:vertical{
height:20px;
transparent;
subcontrol-position:bottom;
border:0px solid;
border-radius: 5px;
}
QScrollBar::sub-line:vertical{
height:20px;
transparent;subcontrol-position:top;
border:0px solid;
border-radius: 5px;
}
QScrollBar::add-page:vertical{
 #e1f0fa;
border:0px solid;border-radius: 5px;
}

QScrollBar::sub-page:vertical{
border:0px solid
;border-radius: 5px;
}
QScrollBar::up-arrow:vertical{
border:0px solid;
border-radius: 3px;
}

QScrollBar::down-arrow:vertical {
border:0px solid;
border-radius: 3px;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="Widget_groupList">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>261</width>
          <height>421</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>411</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayoutGroupList">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="page_space"/>
    </widget>
    <widget class="QLabel" name="label_headImage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
 border-image: url(':/ui/ui/group.png');
 background: rgb(0, 255, 255);
}
QLabel:hover{
	border:2px solid;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_nickname">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color :rgb(85, 255, 255);
font : bold;
font-size : 20px;
</string>
     </property>
     <property name="text">
      <string>昵称</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_email">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color :rgb(85, 255, 255);
font : bold;
font-size : 14px;
</string>
     </property>
     <property name="text">
      <string>email</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_groups">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
 border-image: url(':/ui/groups.png');
}
QPushButton:hover{
	border:2px solid;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_friends">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
 border-image: url(':/ui/friends.png');
}
QPushButton:hover{
	border:2px solid;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_messages">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
 border-image: url(':/ui/messages.png');
}
QPushButton:hover{
	border:2px solid;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page"/>
  </widget>
  <widget class="QWidget" name="widget_bottom" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>700</y>
     <width>301</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 170, 255, 10%)</string>
   </property>
   <widget class="QPushButton" name="pushButton_settings">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
 border-image: url(':/ui/settings.png');
}
QPushButton:hover{
 	border:2px solid;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_addUser">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
 border-image: url(':/ui/add-user.png');
}
QPushButton:hover{
 	border:2px solid;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_addGroup">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
 border-image: url(':/ui/add-group.png');
}
QPushButton:hover{
 	border:2px solid;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_signinGroup">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
 border-image: url(':/ui/signin-group.png');
}
QPushButton:hover{
 	border:2px solid;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="toolButton_min">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{
	border-image: url(':/ui/btn_min.png')
}
QToolButton:hover{
   border:2px solid;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_close">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{
	border-image: url(':/ui/btn_close.png')
}
QToolButton:hover{
   border:2px solid;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
